include/heyoka/kw.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Francesco Biscani (bluescarni@gmail.com), Dario Izzo (dario.izzo@gmail.com)
+//
+// This file is part of the heyoka library.
+//
+// This Source Code Form is subject to the terms of the Mozilla
+// Public License v. 2.0. If a copy of the MPL was not distributed
+// with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef HEYOKA_KW_HPP
+#define HEYOKA_KW_HPP
+
+#include <heyoka/detail/igor.hpp>
+
+namespace heyoka
+{
+
+namespace kw
+{
+
+IGOR_MAKE_NAMED_ARGUMENT(mascon_masses);
```

Comment:
Any particular reason here to have both ``mascon_masses`` and ``masses`` in the kwargs? Couldn't we just use a single ``masses`` kwarg whenever we expect a list of masses?